src/auth/auth-router.js

```diff
@@ -0,0 +1,44 @@
+const express = require('express');
+
+const manager = require('./auth-manager');
+const router = express.Router();
+
+/** hook up the routes **/
+router.get('/', manager.verify, (req, res) => {
+  res.status(201).json({ req.decoded.role }).end();
+});
+
+router.post('/', (req, res) => {
+  manager.login(req.body.email, req.body.password, (err, token) => {
+    if (err) {
+      res.status(400).send(err).end();
+    }
+    else {
+      res.status(201).json({ token: token }).end();
+    }
+  });
+});
+
+router.delete('/', manager.verify, (req, res) => {
+  manager.logout(req.decoded.email, (err) => {
+    if (err) {
+      res.status(400).send(err).end();
+    }
+    else {
+      res.status(201).end();
+    }
+  });
+});
+
+router.post('/create/', (req, res) => {
+  manager.create(req.body.email, req.body.password, req.body.role, (err) => {
+    if (err) {
+      res.status(400).send(err).end();
+    }
+    else {
+      res.status(201).end();
+    }
+  });
+});
+
+module.exports = { router, verify: manager.verify }
```

Comment:
Since ```verify()``` is already provided by ```auth-manager```, and the main purpose of this file is to provide the ```auth-router```, I think that things that need this verification middleware should access it through the ```auth-manager```.

Unless there will be a lot more auth middleware in the future, I think it would be best to keep all the auth methods (including middleware) as part of the auth manager.